common/sky/constants.py

```diff
@@ -0,0 +1,32 @@
+from re import U
+import astropy.units as u
+from astropy.time import Time
+
+"""
+Constants that are used here and there.  Some are Quantities,
+others are just floats. Not all are used.
+
+The planet-coefs are for series expansions for the phase functions
+of the planets, used in predicting apparent magnitude.  See code.
+"""
+
+PI = 3.14159265358979
```

Comment:
Not saying we need to do this, but I was just reading about `typing.Final` and how for values like these constants, it would be good to mark them `Final`.

Example:
```
from typing import Final

PI: Final = 3.14159265358979
```

It doesn't *disallow* reassignment, but it flags it in IDEs with type checking if someone tries to do it.